build/Dockerfile

```diff
@@ -19,4 +19,4 @@ ENV OPERATOR=/usr/local/bin/eunomia-operator
 # install operator binary
 COPY _output/bin/eunomia ${OPERATOR}
 
-ENTRYPOINT ["/usr/local/bin/entrypoint"]
+ENTRYPOINT ["/usr/local/bin/eunomia-operator"]
```

Comment:
With this fix, can we now remove the `command:` field from the yamls?

Reply:
Yes, but I'd rather be explicit about it in the yamls. On the other hand it's not necessary anymore.

Reply:
The yamls already mention an *image* named `eunomia-operator` —  is this not explicit enough?

Reply:
Good point. I'll remove it in next commit